packages/synapse-react-client/src/synapse-client/SynapseClient.ts

```diff
@@ -403,6 +405,26 @@ export const getDownloadFromTableRequest = async (
   )
 }
 
+/**
+ * https://rest-docs.synapse.org/rest/POST/entity/id/table/download/csv/async/start.html
+ */
+export const createTablePfbForDownload = async (
```

Comment:
For Terra integration.

I would have liked to use the OpenAPI-generated client but it doesn't yet have great utilities for using the async job APIs